src/xdai-bots/x-kleros-liquid.js

```diff
@@ -182,6 +182,7 @@ module.exports = async (web3, batchedSend) => {
 
     if (readyForNextPhase) {
       batchedSend({
+        args: [],
```

Comment:
Isn't this redundant?

Reply:
Oh, I was tracking the bug and thought this could the the source of the error, but it wasn't and I forgot to remove it.
Will clean that up.